keyboards/hotdox/left.c

```diff
@@ -0,0 +1,130 @@
+#include "action.h"
```

Comment:
It looks like this is all of the IO Expander code?  

If so, `mcp23017.c` may be a better name for the file. 

(just a suggestion)